core/local/atom/initial_diff.js

```diff
@@ -89,6 +89,10 @@ async function initialState(
   const byInode /*: Map<number|string, Metadata> */ = new Map()
   const docs /*: Metadata[] */ = await opts.pouch.byRecursivePathAsync('')
   for (const doc of docs) {
+    if (metadata.needsRevMigration(doc)) {
+      await metadata.migrateRev(doc, opts)
```

Comment:
Maybe rename to something like `#needsTargetMigration()` & `#migrateTarget()`, same as `sides.rev` vs `sides.target`?